cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/plugin/PluginInstantiator.java

```diff
@@ -257,11 +257,13 @@ public PluginClassLoader getPluginClassLoader(ArtifactId artifactId,
       TypeToken<?> configFieldType = pluginType.resolveType(field.getGenericType());
       Object config = instantiatorFactory.get(configFieldType).create();
 
+      PluginProperties rawProperties = plugin.getProperties();
```

Comment:
nit: move this down. It's more readable to declare variables as close to their usage as possible.